s3watcher/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+get_function_name() {
+    echo $(aws cloudformation list-stack-resources --stack-name $1 \
```

Comment:
Also is this meant to run on TC? How will it have the right credentials to do so?